searches/utils.py

```diff
@@ -26,7 +26,7 @@ def search(school, query, semester):
     )
     return Course.objects.filter(
         Q(school=school) & course_name_contains_query & Q(section__semester=semester)
-    )
+    ).annotate(id_count=Count('id')).order_by("id")
```

Comment:
grouping duplicated course record by course id should be more efficient than doing a whole `distinct` on the entire row